middleware-catalog-module/.idea/misc.xml

```diff
@@ -19,15 +19,20 @@
         <option value="$PROJECT_DIR$/camp/HelloWorldFunction/pom.xml" />
```

Comment:
This files inside .idea directory should not be here. They are on .ignore. You need to delete them in your machine then make a git add, git commit, git push.